types/node-fetch/externals.d.ts

```diff
@@ -0,0 +1,18 @@
+export type Event = any;
+export interface AbortSignal {
```

Comment:
This is great, thanks for moving these out. 

However, I think they should be a declaration instead of an exported type, since we only want to describe the shape that the concrete implementation defined somewhere else should look like.

I think it's enough to replace `export` with `declare`.